add notice about scaling in pd node (#6099) (#6219)
pingcap/docs: scale-tidb-using-tiup.md

>
> You can take similar steps to remove the TiDB and PD node.

> > **Note:**
> >
> > The PD Client in TiKV caches the list of PD nodes. The current version of TiKV has a mechanism to automatically and regularly update PD nodes, which can help mitigate the issue of an expired list of PD nodes cached by TiKV. However, after scaling out PD, you should try to avoid directly removing all PD nodes at once that exist before the scaling. If necessary, before making all the previously existing PD nodes offline, make sure to switch the PD leader to a newly added PD node.
> 

Reviewer: ```suggestion
> **Note:**
>
> The PD Client in TiKV caches the list of PD nodes. 
>
> + In versions before v4.0.3, TiKV cannot automatically and regularly update the cache of the PD node list regularly. Only after the PD leader switches or TiKV restarts to load the latest configuration, the cache can be updated. To avoid the issue of an expired list of PD nodes cached by TiKV, the PD cluster should contain at least one PD node that existed before removing and scaling PD nodes. If this condition is not met, you need to manually perform the PD transfer leader operation to update the PD cache list in TiKV.
>
> + In versions v4.0.3 and later, TiKV has a mechanism to automatically and regularly update PD nodes, which can help mitigate the issue of an expired list of PD nodes cached by TiKV. However, after scaling out PD, you should try to avoid directly removing all PD nodes at once that exist before the scaling. If necessary, before making all the previously existing PD nodes offline, make sure to switch the PD leader to a newly added PD node.
```

>
> You can take similar steps to remove the TiDB and PD node.

> **Note:**
>
> The PD Client in TiKV caches the list of PD nodes. 
>
> + In versions before v4.0.3, TiKV cannot automatically and regularly update the cache of the PD node list regularly. Only after the PD leader switches or TiKV restarts to load the latest configuration, the cache can be updated. To avoid the issue of an expired list of PD nodes cached by TiKV, the PD cluster should contain at least one PD node that existed before removing and scaling PD nodes. If this condition is not met, you need to manually perform the PD transfer leader operation to update the PD cache list in TiKV.
>

> > + In versions v4.0.3 and later, TiKV has a mechanism to automatically and regularly update PD nodes, which can help mitigate the issue of an expired list of PD nodes cached by TiKV. However, after scaling out PD, you should try to avoid directly removing all PD nodes at once that exist before the scaling. If necessary, before making all the previously existing PD nodes offline, make sure to switch the PD leader to a newly added PD node.

Reviewer: ```suggestion
> + In versions v4.0.3 and later, TiKV has a mechanism to automatically and regularly update PD nodes, which can help mitigate the issue of an expired list of PD nodes cached by TiKV. However, after scaling out PD, you should try to avoid directly removing all PD nodes at once that exist before the scaling. If necessary, before making all the previously existing PD nodes offline, make sure to switch the PD leader to a newly added PD node.

```

>
> You can take similar steps to remove the TiDB and PD node.

> **Note:**
>
> The PD Client in TiKV caches the list of PD nodes. 
>

> > + In versions before v4.0.3, TiKV cannot automatically and regularly update the cache of the PD node list regularly. Only after the PD leader switches or TiKV restarts to load the latest configuration, the cache can be updated. To avoid the issue of an expired list of PD nodes cached by TiKV, the PD cluster should contain at least one PD node that existed before removing and scaling PD nodes. If this condition is not met, you need to manually perform the PD transfer leader operation to update the PD cache list in TiKV.

Reviewer 2: ```suggestion
> + In versions earlier than v4.0.3, TiKV does not automatically and regularly update the cache of the PD node list. The cache is updated only after the PD leader switches or TiKV restarts to load the latest configuration. To avoid the issue of an expired list of PD nodes cached by TiKV, the PD cluster should contain at least one PD node that existed before the scaling-in or scaling-out. If this condition is not met, you need to manually perform the PD leader transfer to update the PD cache list in TiKV.
```

>

> > + In versions v4.0.3 and later, TiKV has a mechanism to automatically and regularly update PD nodes, which can help mitigate the issue of an expired list of PD nodes cached by TiKV. However, after scaling out PD, you should try to avoid directly removing all PD nodes at once that exist before the scaling. If necessary, before making all the previously existing PD nodes offline, make sure to switch the PD leader to a newly added PD node.

Reviewer 2: ```suggestion
> + Since v4.0.3, TiKV has a mechanism to automatically and regularly update PD nodes, which can help mitigate the issue of an expired list of PD nodes cached by TiKV. However, after scaling out PD, you should try to avoid directly removing all PD nodes at once that existed before the scaling. If necessary, before making all the previously existing PD nodes offline, make sure to switch the PD leader to a newly added PD node.
```
